### PR TITLE
[WIP] partial output objective function

### PR DESCRIPTION
**Pull request by MisterTea**
I don't know enough about keras/theano to make this an official pull request with confidence, but the idea is that one can pass 0's for unknown outputs without affecting the loss function.  This is especially important for deep q learning where an agent only takes one of the outputted actions, and thus the other actions' error is undefined.  By setting a non-zero error for the action taken and zeroes for the other actions, this loss function should ignore the zeroes and optimize for the non-zero values.

### CODE REVIEW

**Comment by fchollet** [comment]
Sorry, but this would not benefit a sufficiently large number of users to justify inclusion in the codebase.
